packages/InteractiveProfilingTool-Core.package/IPTStats.class/instance/formatGarbageCollectionGeneration.withStats..st

```diff
@@ -0,0 +1,10 @@
+formatting
+formatGarbageCollectionGeneration: aLabel withStats: aDict
+
+	| values |
+	
+	values := { #count. #time. #relativeTime. #averageTime } collect: [ :key | aDict at: key ].
```

Comment:
In this case it may make sense to use `#(..)`-Array. Then you would not need to put "#" in front of every Array element.

Reply:
Danke für die coole Idee 👍 . Wurde umgesetzt.